osc/util/models.py

```diff
@@ -12,7 +12,13 @@
 try:
     from typing import get_origin
 except ImportError:
-    from typing_inspect import get_origin
+    def get_origin(typ):
+        result = getattr(typ, "__origin__", None)
+        bases = getattr(result, "__orig_bases__", None)
+        if bases:
+            result = bases[0]
+        return result
```

Comment:
How about reusing `typing_extensions`?
```suggestion
    from typing_extensions import get_origin
```

Reply:
It doesn't seem to be available on python 3.6 (tested on opensuse/leap:15.5)
```
Python 3.6.15 (default, Sep 23 2021, 15:41:43) [GCC] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from typing_extensions import get_origin
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'get_origin'
```